### PR TITLE
feat(ProjectUI): Disabled CR based on project Group

### PR DESCRIPTION
**Pull request by akapti**
[//]: # (Copyright Bosch.IO GmbH 2020)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

> Please provide a summary of your changes here.

- To enable / disable the `clearing request` feature for `projects` based on project `business unit` (Group).

Issue: https://github.com/eclipse/sw360/issues/1434

### How To Test?

- Set the `Business Unit` (Group) for which the `Clearing Request` should be disabled in properties file (add comma separated `Group` for multiple values).
- property key: `org.eclipse.sw360.disable.clearing.request.for.project.group`
- `CR` should be disabled only for those project which belonged to `Business Unit` which are mentioned in above properties file.
- User should not be able to create the new `CR` for projects belonging to `Business Unit` which are mentioned in properties file.

### Checklist
Must:
- [x] All related issues are referenced in commit messages and in PR

Signed-off-by: Abdul Kapti <abdul.kapti@siemens-healthineers.com>

### CODE REVIEW

**Review by JaideepPalit — CHANGES_REQUESTED** [review]
Reviewed and Tested the PR

1. Without any changes in sw360.properties file, DEPARTMENT GROUP is disabled
2. If CR already created for disabled group. There should still be option to view CR from Project view/list

**Comment by akapti** [comment]
Fixed the mentioned issues.

> 1. Without any changes in sw360.properties file, DEPARTMENT GROUP is disabled

Fixed.

> 2. If CR already created for disabled group. There should still be option to view CR from Project view/list

Fixed. Now User can view the existing Clearing Request for Project belonging to Disabled group, But can't create new CR.

File: frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/common/PortalConstants.java [hunk 1]
```diff
@@ -686,6 +688,7 @@
         SSO_LOGIN_ENABLED = Boolean.parseBoolean(props.getProperty("sso.login.enabled", "false"));
         IS_COMPONENT_VISIBILITY_RESTRICTION_ENABLED = Boolean.parseBoolean(
             System.getProperty("RunComponentVisibilityRestrictionTest", props.getProperty("component.visibility.restriction.enabled", "false")));
+        DISABLE_CLEARING_REQUEST_FOR_PROJECT_WITH_GROUPS = props.getProperty("org.eclipse.sw360.disable.clearing.request.for.project.group", "DEPT");
```
Comment (JaideepPalit) [inline frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/common/PortalConstants.java:691]: Is it required to add a default Department - `Dept` as disabled?

Although it is a rare chance that there will be a Department - `Dept`. 

if it is absolutely necessary , then I am fine with it.

**Review by JaideepPalit — APPROVED** [review]
Code looks fine to me.
Tested successfully

**Pull request merged**